app-doc/gimp-help/gimp-help-2.10.0.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 )
+inherit python-any-r1
+
+DESCRIPTION="GNU Image Manipulation Program help files"
+HOMEPAGE="https://docs.gimp.org/"
+SRC_URI="mirror://gimp/help/${P}.tar.bz2"
+
+LICENSE="FDL-1.2"
+SLOT="2"
+KEYWORDS="~alpha ~amd64 ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+IUSE=""
+
+DEPEND="${PYTHON_DEPS}
```

Comment:
`PYTHON_DEPS` should be a BDEPEND with EAPI-7.

Reply:
Changing th PYTHON_COMPAT to python3_{6,7} results in an error
```
../tools/xml2po.py -p po/ca.po ./gimp-keys.xml > xml/gimp-keys-ca.xml
: python_wrapper_setup: python2 is not supported by python3.6 (PYTHON_COMPAT)

```
It's because of `../tools/xml2po.py` contains first line `#!/usr/bin/env python2`.

If `inherit python-any-r1` is changed to `inherit python-r1` then it's ok. But I don't know is `xml2po.py` then actually running via python3.

upd1: 
I remove python:2.7 from system and then tried to run ebuild gimmp...ebuild again and got an error:
```
../tools/xml2po.py -p po/ca.po ./gimp-keys.xml > xml/gimp-keys-ca.xml
python2: no python-exec wrapped executable found in /usr/lib/python-exec.
```

upd2:
Quick changing `#!/usr/bin/env python2` to `#!/usr/bin/env python2` in `../tools/xml2po.py` results in an arror
```
/bin/mkdir -p xml
../tools/xml2po.py -p po/ca.po ./gimp-keys.xml > xml/gimp-keys-ca.xml
Traceback (most recent call last):
  File "../tools/xml2po.py", line 190, in <module>
    main(sys.argv[1:])
  File "../tools/xml2po.py", line 159, in main
    xml2po_main = Main(default_mode, operation, output, options)
  File "/var/tmp/portage/app-doc/gimp-help-2.10.0/work/gimp-help-2.10.0/tools/xml2po/__init__.py", line 538, in __init__
    self.current_mode = self.load_mode(mode)()
  File "/var/tmp/portage/app-doc/gimp-help-2.10.0/work/gimp-help-2.10.0/tools/xml2po/__init__.py", line 549, in load_mode
    module = __import__('xml2po.modes.%s' % modename, globals(), locals(), ['%sXmlMode' % modename])
  File "/var/tmp/portage/app-doc/gimp-help-2.10.0/work/gimp-help-2.10.0/tools/xml2po/modes/docbook.py", line 201
    print "Ignored tags       : " + repr(test.getIgnoredTags())
                                ^
SyntaxError: invalid syntax
```
So the current script in python2 compatible only and required porting to python3.
The replacing print with print() here results other errors like:
```
../tools/xml2po.py -p po/ca.po ./gimp-keys.xml > xml/gimp-keys-ca.xml
Unable to load mode 'docbook'. Falling back to 'basic' mode with automatic detection (-a).
Traceback (most recent call last):
  File "../tools/xml2po.py", line 190, in <module>
    main(sys.argv[1:])
  File "../tools/xml2po.py", line 173, in main
    xml2po_main.merge(mofile, filenames[0])
  File "/var/tmp/portage/app-doc/gimp-help-2.10.0/work/gimp-help-2.10.0/tools/xml2po/__init__.py", line 593, in merge
    doc.doSerialize(doc.doc)
  File "/var/tmp/portage/app-doc/gimp-help-2.10.0/work/gimp-help-2.10.0/tools/xml2po/__init__.py", line 523, in doSerialize
    outtxt += self.doSerialize(child)
  File "/var/tmp/portage/app-doc/gimp-help-2.10.0/work/gimp-help-2.10.0/tools/xml2po/__init__.py", line 517, in doSerialize
    (starttag, content, endtag, translation) = self.processElementTag(node, repl, True)
  File "/var/tmp/portage/app-doc/gimp-help-2.10.0/work/gimp-help-2.10.0/tools/xml2po/__init__.py", line 436, in processElementTag
    myrepl.append(self.processElementTag(child, myrepl, True))
  File "/var/tmp/portage/app-doc/gimp-help-2.10.0/work/gimp-help-2.10.0/tools/xml2po/__init__.py", line 448, in processElementTag
    translation = self.app.getTranslation(norm_outtxt)
  File "/var/tmp/portage/app-doc/gimp-help-2.10.0/work/gimp-help-2.10.0/tools/xml2po/__init__.py", line 653, in getTranslation
    res = self.gt.ugettext(text.decode('utf-8'))
AttributeError: 'GNUTranslations' object has no attribute 'ugettext'
```